Update Readme.md (#1)
DevExpress-Examples/winforms-create-a-custom-exporter-for-pivotgridcontrol-with-xtrareport: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2231)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # Pivot Grid for WinForms - How to create a custom exporter for the PivotGridControl with the XtraReport suite

Reviewer: ```suggestion
# Pivot Grid for WinForms - How to Create a Custom Exporter for the PivotGridControl with the XtraReport Suite

```

> This example illustrates how to dynamically create a custom report based on [PivotGridControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl) content. This exporter provides the AutoRowHeight, BestFit and FitToPage features.

Reviewer: ```suggestion
This example illustrates how to dynamically create a custom report based on the [PivotGridControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl) content. This exporter supports the AutoRowHeight, BestFit and FitToPage features.

```

![Pivot Grid for WinForms - Custom Exporter](images/custom-exporter-pivot-grid-winforms.png)

> It also implements the following suggestions:

Reviewer: ```suggestion
This example also implements the scenarios introduced in the following suggestions:

```

> - [S130430: Support word wrap when printing and exporting](https://www.devexpress.com/Support/Center/p/S130430) 

Reviewer: ```suggestion
- [Support Word Wrap When Printing and Exporting](https://www.devexpress.com/Support/Center/p/S130430) 

```

> - [S91257: Utilize a Line color when printing](https://www.devexpress.com/Support/Center/p/S91257)

Reviewer: ```suggestion
- [Use a Line Color When Printing](https://www.devexpress.com/Support/Center/p/S91257)

```

> - [AS9011: Print row headers on every page on export/printing](https://www.devexpress.com/Support/Center/p/AS9011)

Reviewer: ```suggestion
- [Print Row Headers on Every Page When Exporting/Printing](https://www.devexpress.com/Support/Center/p/AS9011)

```

> - [S18650: Add an event that allows to provide custom size for field values (MeasureFieldValue, or so) suggestion for printing facilities.](https://www.devexpress.com/Support/Center/p/S18650)

Reviewer: ```suggestion
- [Add an Event that Supplies a Custom Size for Field Values](https://www.devexpress.com/Support/Center/p/S18650)

```

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2231)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Pivot Grid for WinForms - How to Create a Custom Exporter for the PivotGridControl with the XtraReport Suite

> This example illustrates how to dynamically create a custom report based on the [PivotGridControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl) content. This exporter supports the AutoRowHeight, BestFit and FitToPage features.

Reviewer 2: ```suggestion
This example illustrates how to create a custom report dynamically based on the [PivotGridControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl) content. This exporter supports the AutoRowHeight, BestFit, and FitToPage features.
```
